src/lib.rs

```diff
@@ -51,11 +51,14 @@
 None.
 */
 
+#![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
```

Comment:
I *think* the pattern in https://github.com/linebender/xilem/pull/877 is better, because of the interaction with the implicit prelude.

Reply:
I've done this now!